tests/components/homematicip_cloud/helper.py

```diff
@@ -105,19 +137,11 @@ def get_async_home_mock(self):
         and sets reuired attributes.
         """
         mock_home = Mock(
```

Comment:
If we use an `asynctest.Mock` maybe we can avoid having to define the coroutines above. I assume they are present on the spec class?

Reply:
The wraped object is not the async version, because it is not needed and far more complicated to initialize. But i need these coroutines for  service tests.

Reply:
`asynctest.Mock` is the solution.
Great.